pkg/server/container_start.go

```diff
@@ -300,6 +367,20 @@ func addOCIBindMounts(g *generate.Generator, mounts []*runtime.Mount) {
 		// TODO(random-liu): [P1] Apply selinux label
 		g.AddBindMount(src, dst, options)
 	}
+	if privileged {
+		spec := g.Spec()
+		// clear readonly for /sys and cgroup
```

Comment:
Let's at least add unit test for this behavior, and leave out the others. But please add TODO for adding test for device mapping code.